src/test/kotlin/uk/gov/justice/digital/hmpps/personrecord/message/listeners/CourtCaseEventsListenerIntTest.kt

```diff
@@ -68,12 +67,9 @@ class CourtCaseEventsListenerIntTest : IntegrationTestBase() {
       .build()
 
     // when
-    val publishResponse = courtCaseEventsTopic?.snsClient?.publish(publishRequest)?.get()
+    courtCaseEventsTopic?.snsClient?.publish(publishRequest)?.get()
 
     // then
-    assertThat(publishResponse?.sdkHttpResponse()?.isSuccessful).isTrue()
-    assertThat(publishResponse?.messageId()).isNotNull()
-
```

Comment:
These assertions don't seem necessary - the following `await` will block and time out if the publish has failed for any reason